app/models/post.rb

```diff
@@ -0,0 +1,53 @@
+class Post < ActiveRecord::Base
+	#allow accessing this data through the post creation form
+	attr_accessor :tag_string
+
+	#---ASSOCIATIONS----------------------------------------
+	has_many :comments, dependent: :destroy
+	belongs_to :user
+
+	has_many :favorites, dependent: :destroy
+	has_many :users_who_favorited, through: :favorites, source: :user
+
+	has_many :tagizations, dependent: :nullify
+	has_many :tags, through: :tagizations
+
+	#from paperclip
+	#can allow other sizes if wanted later on
+	has_attached_file :picture, :styles => { :medium => "350x350>" }, 
```

Comment:
did you want to add a default image if they don't upload one? also did you want to restrict the file type?
```ruby
  has_attached_file :image, :styles => { :medium => "300x300#", :thumb => "100x100#" }, :default_url => ":style/coffee.jpg"
  validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
```